app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -113,6 +113,7 @@
         android:layout_height="48dp"
         android:layout_marginEnd="4dp"
         android:layout_marginBottom="24dp"
+        android:importantForAccessibility="no"
```

Comment:
Ditto as above comment, I cannot reproduce that talkback is picking image views separately